lib/field-locale.ts

```diff
@@ -44,8 +44,10 @@ export default class FieldLocale implements FieldAPI {
       }
     })
 
-    channel.addHandler('schemaErrorsChanged', errors => {
-      this._schemaErrorsChangedSignal.dispatch(errors)
+    channel.addHandler('schemaErrorsChangedForFieldLocale', (id, locale, errors) => {
+      if (id === this.id && locale === this.locale) {
+        this._schemaErrorsChangedSignal.dispatch(errors)
```

Comment:
This means the other `.dispatch()` call on `_schemaErrorsChangedSignal` can be removed. It'll stay as it is in older versions but from now on this handles all the fields.

Reply:
@jelz I think I've already replaced the previous dispatch logic, unless I'm missing something.

Reply:
Ah sorry I thought it was an addition not replacement 🤦